src/main/java/org/interledger/cryptoconditions/ConditionImpl.java

```diff
@@ -2,17 +2,29 @@
 
 import java.util.EnumSet;
 
+
 import org.interledger.cryptoconditions.Condition;
 import org.interledger.cryptoconditions.ConditionType;
 import org.interledger.cryptoconditions.FeatureSuite;
+import org.interledger.cryptoconditions.encoding.Base64Url;
 
-final class ConditionImpl implements Condition {
+public final class ConditionImpl implements Condition {
```

Comment:
This was is intentionally package private. There is no need for people using this lib to have access to this class it's juts for convenience. They should rather use the interface.